cirq/google/api/v2/calibration.proto

```diff
@@ -0,0 +1,100 @@
+syntax = "proto3";
+
+import "cirq/google/api/v2/metrics.proto";
+import "cirq/google/api/v2/program.proto";
+
+package cirq.google.api.v2;
+
+option java_package = "com.google.cirq.google.api.v2";
+option java_outer_classname = "BatchProto";
+option java_multiple_files = true;
+
+//  This message represents a request to execute a custom calibration routine.
+//
+message FocusedCalibration {
+     // This field represents each invocation of a calibration
+     // procedure.
+     //
+     // For instance, each (unique) moment of a circuit could be
+     // calibrated using parallel_xeb.  In this case,
+     // each moment would have its own CalibrationLayer.
+     //
+     // Some calibrations, such as readout calibration,
+     // will only have one layer.
+     repeated CalibrationLayer layers = 1;
+}
+
+// Each CalibrationLayer represents one invocation
+// of a calibration procedure.
+message CalibrationLayer {
+     // The type of the calibration procedure to execute.
+     // There will be a number of calibration types that can be run,
+     // so this will need to be in a set of acceptable values.
+     string calibration_type = 1;
+
+     // A circuit that identifies the layer or circuit to optimize
+     // if the calibration requires this.  For many calibrations,
+     // this will be a single moment representing the layer to
+     // optimize for.
+     Circuit circuit = 2;
+}
+
+// The results returned by a FocusedCalibration request.
+message FocusedCalibrationResults {
+
+     // The results of each CalibrationLayer request.
+     // There will be one CalibrationLayerResults message for each
+     // CalibrationLayer in the request, and the results will
+     // correspond to the order of the requests.
+     repeated CalibrationLayerResults results = 1;
+}
+
+// Error codes 
+enum CalibrationLayerCode {
+
+     // For backwards compatibility.
```

Comment:
go/protodosdonts#do-include-an-unspecified-value-in-an-enum

Reply:
Gotcha - that makes sense - I think we can expand on that a bit more in this comment and/or link a publicly available link explaining the reason. 

Also from that same paragraph: 

> All enum values declared under a container message are in the same C++ namespace, so prefix the unspecified value with the enum’s name to avoid compilation errors. 

i.e. this should be called `CALIBRATION_RESULT_UNKNOWN` or if we follow the other enums in our protos it might be `CALIBRATION_RESULT_UNSPECIFIED`.